viewer/db.js

```diff
@@ -215,18 +223,22 @@ function fixIndex (index) {
     }).join(',');
   }
 
-  // If prefix isn't there, add it. But don't add it for sessions2 unless really set.
-  if (!index.startsWith(internals.prefix) && (!index.startsWith('sessions2') || internals.prefix !== 'arkime_')) {
-    index = internals.prefix + index;
-  }
+  // Don't fix extra  user-specified indexes from the queryExtraIndices
+  if (!internals.queryExtraIndicesRegex.some(re => re.test(index))) {
```

Comment:
`fixIndex` should leave indices that are user-specified (from `queryExtraIndices`) alone, so we double check for that here. `fixIndex` can be called with either a pattern (`arkime_sessions3-*`) or a specific index name (`arkime_sessions3-240311`) so we use `internals.queryExtraIndicesRegex` here instead of `internals.info.queryExtraIndices` because the regex will match either kind